### PR TITLE
Added ElevateConfirmDialog

### PR DESCRIPTION
**Pull request by yaichenbaum**
For #4036

### CODE REVIEW

File: Files/Dialogs/ElevateConfirmDialog.xaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿<ContentDialog
```
Comment (d2dyno1) [inline Files/Dialogs/ElevateConfirmDialog.xaml:1]: @yair100 You could use DynamicDialog here :eyes:

Comment (d2dyno1) [reply Files/Dialogs/ElevateConfirmDialog.xaml:1]: It'd be easier to maintain

Comment (yaichenbaum) [reply Files/Dialogs/ElevateConfirmDialog.xaml:1]: @d2dyno1 We can do that in another PR, we need to get this merged in order to ship Files in the Microsoft Store.

Comment (d2dyno1) [reply Files/Dialogs/ElevateConfirmDialog.xaml:1]: @yair100 much less code and work! smh
```csharp
DynamicDialog dynamicDialog = new DynamicDialog(new DynamicDialogViewModel()
{
            TitleText = "ElevateConfirmDialog/Title".GetLocalized(),
            PrimaryButtonText = "ElevateConfirmDialog/PrimaryButtonText".GetLocalized(),
            CloseButtonText = "ElevateConfirmDialog/CloseButtonText".GetLocalized(),
            DynamicButtons = DynamicDialogButtons.Primary | DynamicDialogButtons.Cancel
});
```

**Comment by gave92** [comment]
Is it ok if I change it like this?
image

**Comment by yaichenbaum** [comment]
@gave92 Sure!

File: Files/Dialogs/ElevateConfirmDialog.xaml (hunk 1 above)
Comment (yaichenbaum) [reply Files/Dialogs/ElevateConfirmDialog.xaml:1]: @d2dyno1 Does this still make sense to use even though the dialog is going to be used in multiple places?

**Review by gave92 — APPROVED** [review]

**Pull request merged**